apps/src/sites/code.org/pages/public/yourschool.js

```diff
@@ -1,18 +1,25 @@
 import $ from 'jquery';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
 import YourSchool from '@cdo/apps/templates/census2017/YourSchool';
+import {getStore, registerReducers} from '@cdo/apps/redux';
+import isRtl from '@cdo/apps/code-studio/isRtlRedux';
```

Comment:
@islemaster is it bad practice for me to import this "code-studio" reducer on a code.org page?

Reply:
It should be fine.  I think we're trying to eliminate that boundary anyway.